corehq/apps/fixtures/upload/failure_messages.py

```diff
@@ -31,9 +31,9 @@
         "'{prop_key}' is not correctly formatted in 'types' sheet. Whether a field is indexed should be specified "
         "as 'field 1: is_indexed?'. Its value should be 'yes' or 'no'."
     ),
-    "invalid_field_name_numerical": gettext_noop(
+    "invalid_field_name": gettext_noop(
         "Error in 'types' sheet for 'field {i}', '{val}'. "
-        "Field names should be strings, not numbers"
+        "Field names must be valid XML tag names"
```

Comment:
I bet a lot of users won't know what this means. Can this be something like, "Field names should consists of letters, numbers, and underscores"?

Reply:
Do you think it's a problem if the error message states a more restrictive set than what is actually allowed? The set of characters allowed in XML tags is quite large (I could not easily find a regular expression that captures it).

Reply:
Nah, I think it's fine. It'll limit artistic expression a little, but I can't think of a strong reason why a lookup table name would need special characters. Case property names are pretty restricted.